src/analysis/special_functions/exp_log.lean

```diff
@@ -475,6 +475,20 @@ end
 lemma log_nonpos (hx : 0 ≤ x) (h'x : x ≤ 1) : log x ≤ 0 :=
 (log_nonpos_iff' hx).2 h'x
 
+lemma log_nonzero_of_ne_one (x: ℝ) (hx_pos: 0 < x) (hx: x ≠ 1): real.log x ≠ 0 :=
```

Comment:
Do we have the contrapositive (`real.log x = 0 → x = 1`)? I guess that could be proved (untested) as
```lean
  apply log_inj_pos hx_pos zero_lt_one,
  rwa real.log_one,
```

Reply:
Thanks, your suggestion became :
```lean
lemma one_of_pos_of_log_eq_zero {x : ℝ} (h₁: 0 < x) (h₂: real.log x = 0) : x = 1 :=
begin
  apply log_inj_pos h₁ zero_lt_one,
  rwa real.log_one,
end```